tests/testapps/MinApp/MinApp.vsimporter/MinApp-WinStore10/project.json

```diff
@@ -0,0 +1,17 @@
+{
+  "dependencies": {
+    "WinObjC.Frameworks": "*",
```

Comment:
ah, that's right - thanks for the clarification.

---
In reply to: [107011503](https://github.com/Microsoft/WinObjC/pull/2278#discussion_r107011503) [](ancestors = 107011503)